src/magic_folder/cli.py

```diff
@@ -194,8 +177,6 @@ class InitializeOptions(usage.Options):
     """
```

Comment:
Perhaps the docstring needs to be updated with "magic-folder add" instead of "create"?